src/components/policy/policy_external/include/policy/cache_manager_interface.h

```diff
@@ -728,6 +728,14 @@ class CacheManagerInterface {
   virtual ExternalConsentStatus GetExternalConsentStatus() = 0;
 
   /**
+   * @brief Creates externalConsentStatus data structure from policy table
+   section
```

Comment:
@pold500 Minor moment, but please check the doxygen comments after checkstyle fix since they can break in this way.